src/CodeCompliance/Domain/Checks/PrivateApi/Used/Facade.php

```diff
@@ -41,6 +42,14 @@ public function getGuideline(): string
         return 'Please avoid usage of %s(...) in %s';
     }
 
+    /**
+     * @return string
+     */
+    public function getDocumentationUrl(): string
+    {
+        return static::DOCUMENTATION_BASE_URL . 'private-api-is-extended.html';
```

Comment:
not sure, but might be better
https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-extended.html#example-of-code-that-causes-an-upgradability-error-extending-a-private-api-dependency-provider

Reply:
In this case it looks like we need more specific example for this rule check. I create another ticket for that - https://spryker.atlassian.net/browse/PPLUS-1626

And till that I leave a general link in this case if it's fine for you